File: Service_eHospital_DongNai_A/Implement/XML4Service.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class XML4Service : BaseService<XML4, IXML4Repository>
    , IXML4Service
    {
    private readonly IXML4Repository _XML4Repository;
    public XML4Service(IXML4Repository XML4Repository) : base(XML4Repository)
    {
    _XML4Repository = XML4Repository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/XML4.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { XML4 } from './XML4.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class XML4Service extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'XML4_id', 'LoaiKCB', 'IdMaLienKet', 'Loai_KQ', 'IDRef', 'MA_DICH_VU', 'MA_CHI_SO', 'TEN_CHI_SO', 'GIA_TRI', 'MA_MAY', 'MO_TA', 'KET_LUAN', 'NGAY_KQ', 'NgayTao', 'NguoiTao', 'NgayCapNhat', 'NguoiCapNhat'];

    List: XML4[] | undefined;
    ListFilter: XML4[] | undefined;
    FormData!: XML4;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "XML4";
    }
}

